### ISSUE TITLE
If VEP does not produce output, MultiQC fails

### ISSUE THREAD

**Issue by FriederikeHanssen** [ewels/MultiQC#1681]
### Description of bug

Hi!

Whenever we use VEP with inproper data, the tool itself runs fine, but produces incorrect output. If it is then passed to MultiQC, it fails. 
It would be great if MQC would just produce an empty plot instead.

cc @maxulysse 
### MultiQC Error log

```console
/// MultiQC 🔍 | v1.12

|           multiqc | Only using modules: custom_content, fastqc, cutadapt, picard, samtools, qualimap, gatk, bcftools, vcftools, snpeff, vep
|           multiqc | Search path : /Users/monarchy/Projects/Coding/sarek/work/c8/9789c5d3967abee6ddbdfda35c07e5
|         searching | ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 12/12  
|    custom_content | software_versions: Found 1 sample (html)
|    custom_content | nf-core-sarek-summary: Found 1 sample (html)
|               vep | Found 1 VEP summaries
╭────────────────── Oops! The 'vep' MultiQC module broke... ───────────────────╮
│ Please copy this log and report it at                                        │
│ https://github.com/ewels/MultiQC/issues                                      │
│ Please attach a file that triggers the error. The last file found was:       │
│ ./1234N.summary.html                                                         │
│                                                                              │
│ Traceback (most recent call last):                                           │
│   File "/usr/local/lib/python3.10/site-packages/multiqc/multiqc.py", line 65 │
│     output = mod()                                                           │
│   File "/usr/local/lib/python3.10/site-packages/multiqc/modules/vep/vep.py", │
│     self.bar_graph_consequences()                                            │
│   File "/usr/local/lib/python3.10/site-packages/multiqc/modules/vep/vep.py", │
│     if max([len(d) for d in plot_data]) == 0:                                │
│ ValueError: max() arg is an empty sequence                                   │
│                                                                              │
╰──────────────────────────────────────────────────────────────────────────────╯
|           multiqc | Compressing plot data
|           multiqc | Report      : multiqc_report.html
|           multiqc | Data        : multiqc_data
|           multiqc | Plots       : multiqc_plots
|           multiqc | MultiQC complete
~
```

**Comment by ewels** [comment]
Unfortunately I can't access your user space 😉 Can you upload the file instead?

Issue is very similar to https://github.com/ewels/MultiQC/issues/1542 but seems like a different exception type 🤔

**Comment by FriederikeHanssen** [comment]
😵‍💫 sorry, weird it didn't upload the file. it doesn't want to upload the html file. give me a sec

**Comment by lukatop** [comment]
Hi, I can reproduce this issue. If there are no variants in VCF, VEP creates HTML, but MultiQC fails when it tries to parse it.

**Comment by ewels** [comment]
Fixed in d020f9f679592affe421087dbc149014dd37b964 - thanks!
